phpBB/adm/style/acp_email.html

```diff
@@ -19,12 +19,14 @@
 	<legend>{L_COMPOSE}</legend>
 <dl>
 	<dt><label for="group">{L_SEND_TO_GROUP}{L_COLON}</label></dt>
+	<!-- EVENT acp_email_group_options_prepend -->
```

Comment:
This is for adding additional html to implement a kind of the live search functionality (the ext is not mine though).